scripts/release/mule/package/linux/amd64/docker/package.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
+echo
+
+OS_TYPE="$1"
+ARCH="$2"
+WORKDIR="$3"
+
+if [ -z "$OS_TYPE" ] || [ -z "$ARCH" ] || [ -z "$WORKDIR" ]; then
+    echo OS, ARCH and WORKDIR variables must be defined.
+    exit 1
+fi
+
+BRANCH=$(./scripts/compute_branch.sh)
+export BRANCH
+CHANNEL=$(./scripts/compute_branch_channel.sh "$BRANCH")
+export CHANNEL
+
+TARBALLS="./tmp/node_pkgs/$OS_TYPE/$ARCH"
+FULLVERSION=$(./scripts/compute_build_number.sh -f)
+export FULLVERSION
+
+ALGOD_INSTALL_TAR_FILE="$TARBALLS/node_${CHANNEL}_${OS_TYPE}-${ARCH}_${FULLVERSION}.tar.gz"
+
+if [ -f "$ALGOD_INSTALL_TAR_FILE" ]; then
+    echo "using install file $ALGOD_INSTALL_TAR_FILE"
+else
+    echo "error, $ALGOD_INSTALL_TAR_FILE does not exist"
+    exit 1
+fi
+
+INPUT_ALGOD_TAR_FILE="temp_install.tar.gz"
+CHANNEL_VERSION="${CHANNEL}_${FULLVERSION}"
+PKG_DIR="algod_pkg_$CHANNEL_VERSION"
+DOCKER_EXPORT_FILE="algod_docker_export_$CHANNEL_VERSION.tar.gz"
+DOCKER_PKG_FILE="algod_docker_package_$CHANNEL_VERSION.tar.gz"
+DOCKER_IMAGE="algorand/algod_$CHANNEL_VERSION:latest"
+DOCKERFILE=./docker/build/algod.Dockerfile
+START_ALGOD_FILE=./docker/release/start_algod_docker.sh
+ALGOD_DOCKER_INIT=./docker/release/algod_docker_init.sh
+
+echo "building '$DOCKERFILE' with install file $ALGOD_INSTALL_TAR_FILE"
+cp "$ALGOD_INSTALL_TAR_FILE" "/tmp/$INPUT_ALGOD_TAR_FILE"
+cp "$ALGOD_DOCKER_INIT" /tmp
+docker build --build-arg ALGOD_INSTALL_TAR_FILE="$INPUT_ALGOD_TAR_FILE" /tmp -t "$DOCKER_IMAGE" -f "$DOCKERFILE"
+
+mkdir -p "/tmp/$PKG_DIR"
+
+echo "exporting image '$DOCKER_IMAGE' to file '$DOCKER_EXPORT_FILE'"
+docker save --output "/tmp/$PKG_DIR/$DOCKER_EXPORT_FILE" "$DOCKER_IMAGE"
+
+DOCKER_EXPORT_STATUS=$?
+if [ "$DOCKER_EXPORT_STATUS" -ne 0 ]; then
+    echo "Error exporting docker image: $DOCKER_EXPORT_STATUS"
+    exit "$DOCKER_EXPORT_STATUS"
+fi
+
+echo "creating docker package tar file $DOCKER_PKG_FILE"
+cp "$START_ALGOD_FILE" "/tmp/$PKG_DIR/"
+cp ./docker/release/deploy_README.md "/tmp/$PKG_DIR/README.md"
+sed -i "s/%CHANNEL_VERSION%/$CHANNEL_VERSION/g" "/tmp/$PKG_DIR/start_algod_docker.sh"
```

Comment:
if you want it to work on Mac as well, you'd better change it into : 
`sed -i '' "s/%CHANNEL_VERSION%/$CHANNEL_VERSION/g" "/tmp/$PKG_DIR/start_algod_docker.sh"`

( note the two single quotes after the `-i` )

Reply:
I changed to be unix-centric because this will be running in an Ubuntu docker container.